src/proxy.cc

```diff
@@ -1581,8 +1626,10 @@ ncclResult_t ncclProxyCreate(struct ncclComm* comm) {
     proxyState->ncclCollNet = comm->ncclCollNet;
     memcpy(proxyState->buffSizes, comm->buffSizes, sizeof(comm->buffSizes));
 
+    pthread_t thread; 
```

Comment:
Revise name to be related with resilience

Reply:
revised.